frontend/src/features/documents/list/DocumentListView.tsx

```diff
@@ -65,12 +70,36 @@ export const DocumentListView: React.FunctionComponent<IDocumentListViewProps> =
   const [isDetailsVisible, setIsDetailsVisible] = useState(false);
   const [selectedDocument, setSelectedDocument] = useState<Api_Document | undefined>(undefined);
 
-  const handleViewDetails = (values: Api_Document) => {
+  const handleViewDetails = (document: Api_Document) => {
     setIsDetailsVisible(true);
-    setSelectedDocument(values);
+    setSelectedDocument(document);
+  };
+
+  const handleViewDetailsClose = () => {
+    setIsDetailsVisible(false);
+    setSelectedDocument(undefined);
+  };
+  const handleDeleteClicked = (document: Api_Document) => {
```

Comment:
nit: I think we use present tense when naming click handlers; ie "handleDeleteClick"